package.json

```diff
@@ -19,11 +19,7 @@
     "type": "git",
     "url": "git+https://github.com/KineticCafe/actions-dco.git"
   },
-  "keywords": [
-    "github-actions",
-    "dco",
-    "dco-signoff"
-  ],
+  "keywords": ["github-actions", "dco", "dco-signoff"],
```

Comment:
It varies on whether (1) I remember to run `pnpm format` and (2) what version of Biome we are using. I believe that the newest versions now compact where it makes sense, rather than always expanding.